arch/arm/mcount-support.c

```diff
@@ -429,7 +416,9 @@ void mcount_get_stack_arg(struct mcount_arg_context *ctx,
 			if (spec->size <= 4)
 				offset = spec->idx - ARCH_MAX_FLOAT_REGS;
 			else
-				offset = (spec->idx - ARCH_MAX_DOUBLE_REGS) * 2 - 1;
+				offset =
+					(spec->idx - ARCH_MAX_DOUBLE_REGS) * 2 -
+					1;
```

Comment:
I really don't like that, can we suppress such kind of transformation?

Reply:
With the 100 characters limit, we get the following diff:
```diff
-                               offset =
-                                       (spec->idx - ARCH_MAX_DOUBLE_REGS) * 2 -
-                                       1;
+                               offset = (spec->idx - ARCH_MAX_DOUBLE_REGS) * 2 - 1;
```